src/applications/static-pages/burial-how-do-i-apply-widget/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { Provider } from 'react-redux';
+
+export default async (store, widgetType) => {
+  const root = document.querySelector(`[data-widget-type="${widgetType}"]`);
+  if (root) {
+    import(/* webpackChunkName: "burial-how-do-i-apply-widget" */
+    './components/App').then(module => {
```

Comment:
Can we use a regular import here (at the top of the file) instead of a dynamic import?

Reply:
good eyes though bc this is funky normally

Reply:
Thanks for the detail!